lib/App/git/ship/perl.pm

```diff
@@ -324,6 +326,20 @@ sub _timestamp_to_changes {
   $self->abort('Unable to add timestamp to ./%s', $changelog) unless $self->next_version;
 }
 
+sub _travis_configurated {
+  my ($self) = @_;
+  (my $min = $self->config->{min_perl_version} || 10) =~ s/(?:5\.)?([\d\.]+)/$1/;
+  (my $max = $self->config->{max_perl_version} || 26) =~ s/(?:5\.)?([\d\.]+)/$1/;
+  return YAML::Tiny->new({
+    language => 'perl',
+    perl     => [ map { "5.$_" } $min .. $max ],
+    install  => 'cpanm -n --quiet --installdeps --with-develop .',
+    after_success => 'cover -test -report coveralls',
+    sudo          => "false",
+    notifications => { email => "false" },
+  })->write_string;
```

Comment:
We don't need YAML::Tiny, since this is a very simple structure. Just write the YAML in the __DATA__ section instead.